design-documents/asynchronous-import/base-extension.md

```diff
@@ -23,16 +26,15 @@ Path is relative from Magento Root folder
 
 ```
 {
-  "source": {
-      "import_data": "var/catalog_product.csv",
-      "import_type": "local_path",
-      "uuid": "UUID",
-      "format": {
-          "csv_separator": "string",
-          "csv_enclosure": "string",
-          "csv_delimiter": "string",
-          "multiple_value_separator": "string"
-      }
+  "uuid": "UUID",
+  "source_data": {
+    "source_type": "local",
+    "source_data": "var/catalog_product.csv" 
+  } 
+  "format": {
```

Comment:
I think we should have format_type like csv, xml etc.; I do understand we will currently only support csv.
Instead of calling "csv_separator", we can say use "delimiter", which is commonly understood term. And we could also remove "csv_" from beginning.

Reply:
https://www.php.net/manual/en/function.fgetcsv.php

we can also look at some optional parameters in the link above, for instance "escape, enclosure, length"; some of them are very commonly used in file-upload scenarios.

Reply:
About "csv_" - agree here, as we have type defined in URL, we can use just params as "delimiter", "enclosure" and I will add "escape", make sense.
Length I guess its not really required param in this case. Cause we are basically defining how we will read file, which parts of file, etc. I never had a real usage of "length" param in csv import workflows.